src/query/functions-v2/tests/it/scalars/testdata/math.txt

```diff
@@ -117,28 +117,28 @@ output         : 0.5403023058681398
 ast            : tan(1)
 raw expr       : tan(1_u8)
 checked expr   : tan<Float64>(CAST(1_u8 AS Float64))
-optimized expr : 1.5574077246549023_f64
+optimized expr : 1.557407724654902_f64
```

Comment:
Why does the results change?

Reply:
The unit test is not run in one previous PR

Reply:
@sundy-li is it difference between running m1 mac and x86 linux?

Reply:
I do not know, seems the ci works fine with old number.